docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,700 @@
+---
+title: Examples
+description: Learn how to connect to Apify's datacenter proxies from your application with Node.js (axios and got), Python 2 and 3 and PHP using code examples.
+paths:
+    - proxy/datacenter-proxy/nodejs-examples
+    - proxy/datacenter-proxy/python-examples
+    - proxy/datacenter-proxy/php-examples
+    - proxy/datacenter-proxy/examples
+---
+<!-- Watch out! This file is hard to read because our own "marked-tabs" language fences aren"t syntax-highlighted -->
+
+# Connect to datacenter proxies
+
+This page contains code examples for connecting to [datacenter proxies]({{@link proxy/datacenter_proxy.md}}) using [Apify Proxy](https://apify.com/proxy).
+
+If you are building your own Apify [actor]({{@link actors.md}}), below are [examples](#using-the-apify-sdk) specific to the [Apify SDK](https://sdk.apify.com).
+
+See the [connection settings]({{@link proxy/connection_settings.md}}) page for connection parameters.
+
+## [](#using-standard-libraries-and-languages) Using standard libraries and languages
+
+You can find your proxy password on the [Proxy page](https://my.apify.com/proxy) of the Apify app.
+
+> The `username` field is **not** your Apify username.<br/>
+> Instead, you specify proxy settings (e.g. `groups-SHADER+BUYPROXIES94952`, `session-123`).<br/>
+> Use `auto` for default settings.
+
+For examples using [PHP](https://www.php.net/), you need to have the [cURL](https://www.php.net/manual/en/book.curl.php) extension enabled in your PHP installation. See [installation instructions](https://www.php.net/manual/en/curl.installation.php) for more information.
+
+Examples in [Python 2](https://www.python.org/download/releases/2.0/) use the [six](https://pypi.org/project/six/) library. Run `pip install six` to enable it.
+
+### [](#single-request-with-a-random-ip-address) Single request with a random IP address
+
+The IP address is chosen from all available proxy groups.
+
+Select this option by setting the `username` parameter to `auto`.
+
+```marked-tabs
+<marked-tab header="Node.js (axios)" lang="javascript">
+const HttpsProxyAgent = require("https-proxy-agent");
+const axios = require("axios");
+
+const httpsAgent = new HttpsProxyAgent({
+    host: "proxy.apify.com",
+    port: "8000",
+    // Replace <YOUR_PROXY_PASSWORD> below with your password
+    // found at https://my.apify.com/proxy
+    auth: "auto:<YOUR_PROXY_PASSWORD>"
+});
+
+axiosHttpsAgent = axios.create({ httpsAgent });
+
+async function useProxy() {
+    const response = await axiosHttpsAgent.get("https://api.apify.com/v2/browser-info");
+    console.log(response.data)
+};
+useProxy();
+</marked-tab>
+
+
+<marked-tab header="Node.js (got)" lang="javascript">
+const got = require("got");
+const HttpProxyAgent = require("http-proxy-agent");
+const HttpsProxyAgent = require("https-proxy-agent");
```

Comment:
With `axios` we're only using the `HttpsProxyAgent`. Why is this different?

Reply:
I had to ask @petrpatek for help with this one and it looks like he's more thorough than me :) 
Since the `http` and `https` agents work the same, I'll just keep the `https` one